### ISSUE TITLE
Brushes dont work.

### ISSUE THREAD

**Issue by dhhdfisfhsih** [opentoonz/opentoonz#1838]
The program is working exept brushes, they are dont work, when I try to draw, does not react and puts only points, sometimes lines,and sometimes even draw, but glitchy, from the moment I start to draw , raster doesnt work either, and when i try to draw smth in raster it doesnt work, and then vector stop draw at all. PLS HELP. ( AND yeah i tried in Morevna and Regular OP, same effect)
Tablet is OK, because in other programs it does work. Windows 8. 64 bit (if u need this Information)
I used this program before, it worked rly good, maybe its because of new version . I DONT KNOW

**Comment by dhhdfisfhsih** [comment]
Yep, i can ok draw with mouse, but cant on tablet , by the way mouse still doesnt draw smth of raster brushes

**Comment by dhhdfisfhsih** [comment]
Oh another one, i can draw sharp, coz it doesnt work when i do , it works when i draw really carefully

**Comment by manongjohn** [comment]
Sounds like you have extreme brush lag. What video card do you have?

**Comment by dhhdfisfhsih** [comment]
NVIDIA

**Comment by manongjohn** [comment]
@dhhdfisfhsih My guess is OT isn't using your GPU and running extremely slow. Try going into your NVIDIA control panel and create a 3D profile for OT 1.2..

Is it possible you did this before for 1.1 and forgot? That could explain why it was working before the upgrade. Unfortunately, upgrading to 1.2 changed the folder path and executable name so NVIDIA needs to know the new foldeder/name or it won't utilize your GPU.

**Comment by dhhdfisfhsih** [comment]
it isnt working

**Comment by artisteacher** [comment]
You might want to try renaming the stuff folder and then reinstalling. If there’s a problem with the install itself (perhaps an incompatibility between versions) that can make a difference. If it doesn’t resolve your issue, at least we can rule that out. 

If it’s not a gpu or install problem and you have the time, you could try some of the nightly builds. The latest nightly has all of the most recent fixes. Older versions can help narrow down when an issue started.

**Comment by dhhdfisfhsih** [comment]
can u pls give the link on this nightly build

**Comment by jpturcotte** [comment]
https://github.com/opentoonz/opentoonz_nightlies/releases
